src/commands/autoUpdate.js

```diff
@@ -16,25 +16,22 @@ type Result = {
 
 const cmd: Command<Input, Result> = createCommand('main:autoUpdate', () =>
   Observable.create(o => {
-    const { autoUpdater } = require('electron-updater')
+    const { autoUpdater } = require('@ledgerhq/electron-updater')
 
     const sendStatus = (status, payload) => {
       o.next({ status, payload })
     }
 
-    const handleDownload = async _ => {
+    const handleDownload = async info => {
       try {
         sendStatus('checking')
-        // const appUpdater = await createElectronAppUpdater({
-        //   feedURL: UPDATE_CHECK_FEED,
-        //   updateVersion: info.version,
-        // })
-        // await appUpdater.verify()
+        const appUpdater = await createElectronAppUpdater({ feedURL: UPDATE_CHECK_FEED, info })
+        await appUpdater.verify()
         sendStatus('check-success')
       } catch (err) {
-        // don't throw if the check fail for now. it's a white bullet.
+        logger.critical(err)
+        // don't throw if the check fail for now. it's a blank bullet.
         if (UPDATE_CHECK_IGNORE) {
```

Comment:
does this mean we'll need to set this as true for the first build ?

Reply:
`UPDATE_CHECK_IGNORE` is already set as `true` (since #1705).
We'll need to toggle it to `false` in release n+2, if the update from release n to n+1 didn't raise any issue.